Sources/Action/UIKitExtensions/UIBarButtonItem+Action.swift

```diff
@@ -37,5 +38,28 @@ public extension Reactive where Base: UIBarButtonItem {
             }
         }
     }
+    public func bindToAction <Input,Output>(_ action:Action<Input,Output>?,_ inputTransform: @escaping (Base) -> (Input))   {
```

Comment:
Also, this calls is not in line with Swift 3 - parameters need to be named and a proper Swift-3-y syntax would be `bindTo(action: Action<Input,Output>, transform: @escaping (Base) -> (Input))`